SmartDeviceLinkTests/DevAPISpecs/TestSystemCapabilityObserver.h

```diff
@@ -17,7 +17,9 @@ NS_ASSUME_NONNULL_BEGIN
 @property (assign, nonatomic) NSUInteger selectorCalledCount;
 
 - (void)capabilityUpdated;
-- (void)capabilityUpdatedWithNotification:(SDLSystemCapabilityManager *)capabilityManager;
+- (void)capabilityUpdatedWithCapability:(SDLSystemCapability *)capability;
+- (void)capabilityUpdatedWithCapability:(SDLSystemCapability *)capability error:(NSError *)error;
+- (void)capabilityUpdatedWithCapability:(SDLSystemCapability *)capability error:(NSError *)error subscribed:(BOOL)subscribed;
```

Comment:
Should add a test case for checking the `subscribed` value for `SDLSystemCapabilityTypeDisplays` since that seems to be a unique case.

Reply:
Fixed